dotcom-rendering/src/web/components/NumberedTitleBlockComponent.tsx

```diff
@@ -34,12 +32,7 @@ export const NumberedTitleBlockComponent = ({
 	html,
 	format,
 }: Props) => {
-	const dcrFormat = {
-		display: decideDisplay(format),
-		design: decideDesign(format),
-		theme: decideTheme(format),
-	};
-	const palette = decidePalette(dcrFormat);
+	const palette = decidePalette(decideFormat(format));
```

Comment:
Just checking, are we doing `decideFormat(format)` here because `format` is really `CAPIFormat`?